### DISCUSSION TITLE
uint ethPrice=getPriceUsd()

### DISCUSSION THREAD

**Discussion by Vodvalex** [Cyfrin/foundry-full-course-cu#2283] [Q&A] [answered]
Hi Everyone
In the FundMe.sol or Library PriceConverter I see the following:
uint ethPrice=getPriceUsd()

As far as I understand the latter one is a name of the function: function getPriceUsd().....
My question is is it normal in Solidity that a variable to be  equalled to a name of any function or it is just a special case? 
If yes, then I would appreciate if anyone could give me  a link to the source of  this kind of method or approach in different contracts or explain me what the above mentioned is based on. I could not find this kind of approach  at all.  Thank you.

**Comment by EngrPips** [comment]
Hello @Vodvalex, I don't understand your question. Can you elaborate more?

**Comment by urosognjenovic** [comment, marked as answer]
The case here is that the return value of the function ```getPriceUsd()``` is assigned to the variable ```ethPrice```. There's no such thing as equalizing a variable to a function.

**Reply by Vodvalex** [reply]
Yes I am getting it now  ''**assigned** to the variable ethPrice''.Thanks for clarification.

**Comment by EngrPips** [comment]
Oh interesting.
